README.md

```diff
@@ -32,3 +33,53 @@ class AppKernel
 #Usages
 
 You just have to continue to use former url generation, nothing changes concerning the implementation. The only changes is that you don't have to repeat current route parameters anymore
```

Comment:
"You just have to continue to use former url generation" => "Just continue to use former url generation"